### PR TITLE
feat: add crowdin workflows

### PR DESCRIPTION
**Pull request by raisedadead**
This PR should copy over the Crowdin workflows. I am sure we have to do more stuff on Crowdin side.

### CODE REVIEW

File: .github/crowdin-download.docs.yml [hunk 1]
```diff
@@ -0,0 +1,265 @@
+name: i18n - Download Docs
+on:
+  workflow_dispatch:
+  schedule:
+    # runs Monday and Wednesday at 12:15 PM UTC
+    - cron: '15 12 * * 1,3'
```
Comment (naomi-lgbt) [inline .github/crowdin-download.docs.yml]:
```suggestion
  # schedule:
    # runs Monday and Wednesday at 12:15 PM UTC
    # - cron: '15 12 * * 1,3'
```

File: .github/crowdin-upload.docs.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: i18n - Upload Docs
+on:
+  workflow_dispatch:
+  schedule:
+    # runs every weekday at 11:00 AM UTC
+    - cron: '0 11 * * 1-5'
```
Comment (naomi-lgbt) [inline .github/crowdin-upload.docs.yml]:
```suggestion
  # schedule:
    # runs every weekday at 11:00 AM UTC
    # - cron: '0 11 * * 1-5'
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
We should disable it for now, so we can merge it but not cause a bunch of run failures until I get everything configured on the crowdin end.

I won't be able to set Crowdin itself up until we've landed on a stack we want to use - need to see what file format we'll be using and what scripts I need to write to hide the necessary strings.

**Comment by raisedadead** [comment]
> I won't be able to set Crowdin itself up until we've landed on a stack we want to use

We wont have a breaking change. We will continue with docsify for a bit.

**Review by naomi-lgbt — APPROVED** [review]
Should be good to go. I'll make a subsequent PR to run this on my test project and confirm the action config doesn't need changing, and then follow up with a PR to re-enable the CRON.

**Pull request merged**